### ISSUE TITLE
Update peer dependencies

### ISSUE THREAD

**Issue by timeimp** [microlinkhq/react-json-view#53]
Yarn reports that React 19 doesn't have a dependency resolution with the `8.3.2` version of `react-textarea-autosize` package. 

Version `8.5.6` now lists React 19 as a `peerDependency` making this problem fixed. 

Would it be possible to update the peer dependencies inside `react-json-view` to their latest versions?

**Comment by Kikobeats** [comment]
Yes! Can you send a PR? <picture data-single-emoji=":pra:" title=":pra:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/pra/b3935c9f40d8c090.png" alt=":pra:" align="absmiddle"></picture>

**Comment by andreykovalenko** [comment]
raised the PR to fix it https://github.com/microlinkhq/react-json-view/pull/56

**Comment by Kikobeats** [comment]
Resolved by https://github.com/microlinkhq/react-json-view/pull/57
